src/views/Home/components/WinSection/LotteryCardContent.tsx

```diff
@@ -26,8 +27,17 @@ const LotteryCardContent = () => {
   const { observerRef, isIntersecting } = useIntersectionObserver()
   const [loadData, setLoadData] = useState(false)
   const [lotteryId, setLotteryId] = useState<string>(null)
-  const [currentLotteryPrize, setCurrentLotteryPrize] = useState<BigNumber>(null)
   const cakePriceBusdAsString = usePriceCakeBusd().toString()
+  const { data: currentLotteryPrize = null } = useSWR(
+    lotteryId ? ['lottery', 'currentLotteryPrize', cakePriceBusdAsString] : null,
```

Comment:
cake price changed a lot, maybe not worth to add this into key

Reply:
Done